pkgs/tools/admin/aws-gate/default.nix

```diff
@@ -0,0 +1,54 @@
+{ lib
+, fetchFromGitHub
+, installShellFiles
+, python3
+}:
+
+python3.pkgs.buildPythonApplication rec {
+  pname = "aws-gate";
+  version = "0.11.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "xen0l";
+    repo = pname;
+    rev = version;
+    hash = "sha256-9w2jP4s1HXf1gYiXX05Dt2iXt0bR0U48yc8h9T5M+EQ=";
+  };
+
+  nativeBuildInputs = [
+    python3.pkgs.setuptools
+    python3.pkgs.wheel
+    installShellFiles
+  ];
+
+  propagatedBuildInputs = with python3.pkgs; [
+    pkgs.ssm-session-manager-plugin
+    marshmallow
+    requests
+    boto3
+    pyyaml
+    wrapt
+    cryptography
+    # Should probably patch out the bootstrap command that requires this dependency as we bundle the session manager plugin anyway
```

Comment:
That's probably a good idea if you don't want to maintain `unix-ar`. When looking at the `aws-gate` source, it seems that the whole `aws_gate.bootstrap` module only is used in tests and for invoking the `bootstrap` function here:

https://github.com/xen0l/aws-gate/blob/52f7281af2805e28d44ee3906b14d48acb5201fb/aws_gate/cli.py#L271

So in theory, it should be possible to just remove the whole `bootstrap` subcommand from the `ArgumentParser` and remove all related modules (and tests) and it should work fine.

Reply:
I am not opposed to maintaining `unix-ar`, doubt there will be more versions to maintain in the future and I have found quite a few repositories on github referencing it. When those eventually make their way to nixpkgs, having the dependency here already would ease their maintainers' work.

Still believe the dependency should be removed from `aws-gate` though; do I make a patch to remove the command completely or should I keep the command and have it simply print that the package installed from nixpkgs will always have the plugin? 
I'm leaning towards the latter since it would not perturb user-facing functionality. However, there I am not sure how things are usually done when porting such packages to nix so I defer to your judgment.

Reply:
Got that working and in the process also found that the `requests` dependency could also be removed if no bootstrapping is needed 🥳